src/VisualStudio/LiveShare/Impl/CustomProtocol/RoslynMethods.cs

```diff
@@ -8,6 +8,5 @@ internal static class RoslynMethods
     {
         public const string ProjectsName = "roslyn/projects";
```

Comment:
this is still required for typescript.  Once they switch to their LSP editor (~16.8ish) this method can also be removed.